Guide/src/reference/architecture/openhcl.md

```diff
@@ -7,71 +7,60 @@
 
 * * *
 
-> This page is under construction
-
 ## Overview
 
-The following diagram offers a brief, high-level overview of the OpenHCL
-Architecture.
+OpenHCL is a paravisor execution environment that runs within the guest partition of a virtual machine. It provides virtualization services to the guest OS from within the guest itself, rather than from the host.
+
+The following diagram offers a brief, high-level overview of the OpenHCL Architecture.
 
 ![OpenHCL High Level Overview](./_images/openhcl.png)
 
-## VTLs
+## Virtual Trust Levels (VTLs)
 
-OpenHCL currently relies on Hyper-V's implementation of [Virtual Trust Levels]
-(VTLs) to implement the security boundaries necessary for running OpenVMM as a
-paravisor.
+OpenHCL relies on [Virtual Trust Levels] (VTLs) to establish a security boundary between itself and the guest OS.
 
-VTLs can be backed by:
+- **VTL2:** OpenHCL runs here[^sk]. It has higher privileges and is isolated from VTL0.
+- **VTL0:** The Guest OS (e.g., Windows, Linux) runs here. It cannot access VTL2 memory or resources.
```

Comment:
Maybe include VTL1 in this list instead of as a footnote, since from our perspective they're both just 'guest' VTLs.